src/main/java/it/pagopa/pn/delivery/pnclient/externalregistries/PnExternalRegistriesClientImpl.java

```diff
@@ -1,34 +1,26 @@
 package it.pagopa.pn.delivery.pnclient.externalregistries;
 
-import it.pagopa.pn.delivery.PnDeliveryConfigs;
-import it.pagopa.pn.delivery.generated.openapi.clients.externalregistries.ApiClient;
-import it.pagopa.pn.delivery.generated.openapi.clients.externalregistries.api.InternalOnlyApi;
-import it.pagopa.pn.delivery.generated.openapi.clients.externalregistries.api.PaymentInfoApi;
-import it.pagopa.pn.delivery.generated.openapi.clients.externalregistries.model.PaGroup;
-import it.pagopa.pn.delivery.generated.openapi.clients.externalregistries.model.PaymentInfo;
+import it.pagopa.pn.delivery.generated.openapi.msclient.externalregistries.v1.api.InternalOnlyApi;
+import it.pagopa.pn.delivery.generated.openapi.msclient.externalregistries.v1.api.PaymentInfoApi;
+import it.pagopa.pn.delivery.generated.openapi.msclient.externalregistries.v1.model.PaGroup;
+import it.pagopa.pn.delivery.generated.openapi.msclient.externalregistries.v1.model.PaymentInfo;
+import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Component;
-import org.springframework.web.client.RestTemplate;
 
 import java.util.Collections;
 import java.util.List;
 
 @Slf4j
 @Component
+@RequiredArgsConstructor
 public class PnExternalRegistriesClientImpl {
 
     private final PaymentInfoApi paymentInfoApi;
     private final InternalOnlyApi internalOnlyApi;
 
-    public PnExternalRegistriesClientImpl(@Qualifier("withTracing") RestTemplate restTemplate, PnDeliveryConfigs cfg) {
-        ApiClient newApiClient = new ApiClient( restTemplate );
-        newApiClient.setBasePath( cfg.getExternalRegistriesBaseUrl() );
-        this.paymentInfoApi = new PaymentInfoApi( newApiClient );
-        this.internalOnlyApi = new InternalOnlyApi( newApiClient );
-    }
 
-    public PaymentInfo getPaymentInfo( String paTaxId, String noticeNumber ) {
+    public PaymentInfo getPaymentInfo(String paTaxId, String noticeNumber ) {
         return paymentInfoApi.getPaymentInfo( paTaxId, noticeNumber );
```

Comment:
Qui, ma in generale sui client, manca il log di invokingExternalService https://pagopa.atlassian.net/wiki/spaces/PN/pages/708218354/Linee+guida+Log+applicativi+Internal#Quando-inserire-i-logs-(by-CustomLog)---comprende-log-automatici-e-manuali